lab_4_retrieval_w_clustering/main.py

```diff
@@ -81,6 +110,21 @@ def _calculate_bm25(self, tokenized_document: list[str]) -> Vector:
         Returns:
             Vector: BM25 vector for document.
         """
+        if (not tokenized_document or
+                not isinstance(tokenized_document, list) or
+                not all(isinstance(elem, str) for elem in tokenized_document)):
+            raise ValueError('Invalid arguments')
+
+        calculated_bm25 = [0.0] * len(self._vocabulary)
+        doc_len = len(tokenized_document)
+
+        bm25 = calculate_bm25(self._vocabulary, tokenized_document,
+                              self._idf_values, 1.5, 0.75, self._avg_doc_len, doc_len)
```

Comment:
do not set `1.5, 0.75` - it is argument by default

Reply:
done